app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @public_recipes = public_recipes.all
```

Comment:
This part of the code isn't working, you're supposed to get all the Recipes that have the public attribute true, you could replace it with `@public_recipes = Recipe.where(public: true)` and then display all these Recipes on index page following the wireframe